appium/webdriver/gsm_call_actions.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
What about creating files in `gsm` directory like below?

```
appium/webdriver/extentions/gsm/call_actions.py
```

Or define the class in appium/webdriver/extensions/gsm.py

```
class Gsm(webdriver.Remote):
    ....

class GsmCallActions(object):
   ...
```

Reply:
make sense, I did the latter one.